scope.go

```diff
@@ -400,6 +401,31 @@ func (scope *Scope) ApplyToEvent(event *Event, hint *EventHint) *Event {
 		}
 	}
 
+	// Apply the trace context to errors if there is a Span on the scope. If
+	// there isn't then fall back to the propagation context.
+	if event.Type != transactionType {
```

Comment:
Could we also use this for transactions, so it's centralized?

Reply:
Yeah, looking at the code now that makes sense. I'll make the change. :)